### DISCUSSION TITLE
build:wait fails on approved changed

### DISCUSSION THREAD

**Discussion by Alecktos** [percy/cli#934] [Ideas] [answered]
Hi
I'm using `percy build:wait --project xxxx/ccc --commit HEAD --fail-on-changes` to validate if our Cypress tests (containing percy snapshots) has run successfully in Jenkins. The problem I'm facing is if the changes has been approved in previous build, build:wait still fails. 

Would be good to have the option to not fail on approved changes, or some other command/flag to retrieve if the build has been approved or not.

**Comment by AaronBuxbaum** [comment]
https://github.com/percy/cli/pull/1205 I think this should do it

**Comment by itsjwala** [comment, marked as answer]
[v1.21.0](https://github.com/percy/cli/releases/tag/v1.21.0) is released with the functionality to include `--pass-if-approved` flag alongside `--fail-on-changes` which won't exit CLI if build is approved.
